docker/Dockerfile.devel

```diff
@@ -65,20 +65,36 @@ RUN CMAKE_VERSION=3.17 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install && \
     rm -rf /cmake-${CMAKE_BUILD}
 
-RUN pip install --force-reinstall --extra-index-url https://developer.download.nvidia.com/compute/redist/nightly nvidia-dali-nightly-cuda110
-
 WORKDIR /dali
 
+ARG DALI_DOWNLOAD_EXTRA_INDEX_URL=https://developer.download.nvidia.com/compute/redist/nightly
```

Comment:
Maybe you can have a variable called `flavor` and:
```
ARG DALI_FLAVOR=nightly
ARG DALI_DOWNLOAD_EXTRA_INDEX_URL=https://developer.download.nvidia.com/compute/redist/${DALI_FLAVOR}
ARG DALI_DOWNLOAD_PKG_NAME=nvidia-dali-${DALI_FLAVOR:+${DALI_FLAVOR}-}cuda110
```

Reply:
Just an idea.

Reply:
We decided to keep cmake simpler and have only package name as argument. The argument similar to this flavor is present in the gitlab yaml, though.

Reply:
Understood